src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -171,6 +172,7 @@ const TrackPanelGene = (props: TrackPanelGeneProps) => {
         visibilityIconHelpText={getVisibilityIconHelpText(geneVisibilityStatus)}
         onShowMore={onShowMore}
         toggleExpand={toggleExpand}
+        hideToggle={hideToggle}
       >
```

Comment:
Mmm. A `GroupTrackPanelItemLayout` component is like a `SimpleTrackPanelItemLayout` component, but with two extra parameters: `isCollapsed`, and `toggleExpand`. So if we don't want to show the chevron for genes with a single transcript, why don't we use the `SimpleTrackPanelItemLayout` component for such cases in `TrackPanelGene`? It would keep the purpose of these two components clearly separated instead of blurring the distinction between them.